smt/utils/neural_net/model.py

```diff
@@ -221,9 +221,9 @@ def train(
 
                 # Compute average cost and print output
                 avg_cost = np.mean(optimizer.cost_history).squeeze()
-                self._training_history["epoch_" + str(e)]["batch_" + str(b)] = (
-                    optimizer.cost_history
-                )
+                self._training_history["epoch_" + str(e)][
+                    "batch_" + str(b)
+                ] = optimizer.cost_history
```

Comment:
black again?

Reply:
no "ruff format"

Reply:
ruff has been updated to 3.6 last week and to 3.7 this week.

Reply:
ruff has been updated to 3.6 last week and to 3.7 this week.